Makefile

```diff
@@ -404,7 +404,7 @@ tool-remove-execution-fork: docker-build-remove-execution-fork
 # Check if the go version is 1.13 or higher. flow-go only supports go 1.13 and up.
 .PHONY: check-go-version
 check-go-version:
-	go version | grep 1.13 || go version | grep 1.14 || go version | grep 1.15
+	go version | grep 1.1[3456]
```

Comment:
This won't work, for using grep regex you need to use the values inside single quotation marks.

Reply:
It definitely works.

```
$ echo 1.16 | grep 1.1[3456]
1.16
```

Reply:
It might work on your system, but it is actually dependent on shell. for example zsh will return this 
```
echo 1.16 | grep 1.1[3456]
zsh: no matches found: 1.1[3456]
```